lib/src/main/java/com/auth0/msg/KeyJar.java

```diff
@@ -0,0 +1,4 @@
+package com.auth0.msg;
+
```

Comment:
Please add class level Javadoc
Aren't you going to extend from auth0 class? If so, please add` extends`